pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -375,7 +375,7 @@ func (pe *StandardSecurityPolicyEnforcer) EnforceOverlayMountPolicy(containerID
 	for _, i := range matchedContainers {
 		existing := pe.ContainerIndexToContainerIds[i]
 		if len(existing) >= len(matchedContainers) {
-			errmsg := fmt.Sprintf("layerPaths '%v' already used in maximum number of container overlays", layerPaths)
+			errmsg := fmt.Sprintf("layerPaths '%v' already used in maximum number of container overlays. This is likely because the container was restarted and container restarts are not yet supported.", layerPaths)
```

Comment:
supporting restart by default is coming soon. Maybe we should say that the "This is likely because the security policy allows the container to be run only once" or similar.

Reply:
and by the way, we'll drop support for JSON policy in favor of Rego soon. @matajoh what change needs to happen on the Rego side of things to make this error message more descriptive?

Reply:
The Rego system will not have this limitation, so I don't think this particular error message will be needed. At some point we will add it back in (as it may be desirable for some scenarios) and when we do so we'll ensure the message makes it clear what has happened.

Reply:
That's great news! This PR was just in response to some difficulties Bryce and I had earlier this week trying to figure out why this error message was happening, thanks all